src/app/view/components/main-legacy/main-legacy.component.ts

```diff
@@ -53,16 +33,41 @@ export class MainComponentLegacy implements OnInit, OnDestroy {
      * @param {*} event
      * @memberof DotNavigationComponent
      */
-    @HostListener('document:click', ['$event'])
+    @HostListener('click', ['$event'])
```

Comment:
Turns out that the document.click didn't work in the actual device (I tested in an old iPad 3 I have), so because the component is ALL the page we can listen to the click in the actual component and that worked.